### PR TITLE
Wk18 Malin Vannesjö

### PR DESCRIPTION
**Pull request by malinvan**
https://malins-wk18-api.herokuapp.com

### CODE REVIEW

**Review by leelalaim — APPROVED** [review]
Hey! Your code is nice and clean, have nothing to comment about. Well done! :)

File: README.md [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
+Deploying to Heroku is pretty straight forward but setting up the connection to Atlas was a bit more tricky.
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+My main problem was that due to circumstances I didn't have time to make a frontend for this project or to dive that deep. I would have loved to make a frontend but also to have added more to the API and just to play around with it in general for the sake of learning.
+I plan on doing that as soon as I have the time for it though.
 
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+https://malins-wk18-api.herokuapp.com
```
Comment (KarolCislowski) [inline README.md:17]: short documentation would be nice 🤓

File: server.js [hunk 2]
```diff
@@ -25,15 +39,67 @@ const app = express()
...
+})
+
+app.get('/content', async (req, res) => {
+  let data
+  const { country } = req.query
+  const { genre } = req.query
+  const { releaseYear } = req.query
+  const { title } = req.query
+  const { director } = req.query
+  const { cast } = req.query
```
Comment (KarolCislowski) [inline server.js:57]: It can be done in one line 🤓

File: server.js [hunk 3]
```diff
@@ -25,15 +39,67 @@ const app = express()
...
+      data = await NetflixData.find({ 
+        director: { $regex: director, $options: "i" }
+      })
+    } else if (cast) {
+      data = await NetflixData.find({ 
+        cast: { $regex: cast, $options: "i" }
+      })
+    } else {
+      data = await NetflixData.find()
+    }
```
Comment (KarolCislowski) [inline server.js:84]: When you give your user many query options to filter your data with, you need to provide logic for every combination of them.

**Review by KarolCislowski — APPROVED** [review]
Nice API 👍🏻

**Pull request closed without merging**